app/scripts/views/sign_in.js

```diff
@@ -149,6 +149,11 @@ define(function (require, exports, module) {
       } else if (AuthErrors.is(err, 'INCORRECT_PASSWORD')) {
         return this.showValidationError(this.$('#password'), err);
       }
+
+      if (AuthErrors.is(err, 'EMAIL_HARD_BOUNCE') || AuthErrors.is(err, 'EMAIL_SENT_COMPLAINT')) {
```

Comment:
Should this be done [in the signin-mixin](https://github.com/mozilla/fxa-content-server/blob/f9f7f10d4310163ffc256d1536322d470dcfe646/app/scripts/views/mixins/signin-mixin.js#L70) so that this error is also handled on /force_auth and /signup if the user does signin from signup?